rel/overlay/etc/default.ini

```diff
@@ -106,8 +106,8 @@ couch = couch_bt_engine
 ;couch_server = false
 
 [cluster]
-q=2
-n=3
+q = 2
```

Comment:
Avoid unrelated changes in the PR. We can make a separate PR for formatting

Reply:
I will pay attention next time.